src/core/ThemeProvider/ThemeProvider.test.tsx

```diff
@@ -41,6 +41,11 @@ it('should set light theme', () => {
   expectLightTheme();
 });
 
+it('should set light theme specifying owningDocument', () => {
+  render(<ThemeProvider theme='light' owningDocument={document} />);
```

Comment:
I think this test can be improved to actually use a mock document that is not the same as `document`.

Reply:
updated

Reply:
oh the test is failing now because the mock document doesn't have `body`.

Might need to do something like: 
```ts
const { document } = (new JSDOM(`...`)).window;
```

Reply:
yep, just saw that, will fix.

Reply:
I could not get JSDOM to work, so I opted for an approach that did not require a new import.